web/src/main/java/org/springframework/security/web/webauthn/authentication/WebAuthnAuthenticationRequestToken.java

```diff
@@ -30,6 +32,9 @@
  */
 public class WebAuthnAuthenticationRequestToken extends AbstractAuthenticationToken {
```

Comment:
Because this is already marked as serializable, by virtue of it extending `AbstractAuthenticationToken`, we need to make this change in `6.4.x`.